core/src/main/java/net/sourceforge/jnlp/util/XDesktopEntry.java

```diff
@@ -451,11 +451,11 @@ private NonFileProtocolException(String unable_to_cache_icon) {
      */
     private void cacheIcon() throws IOException, NonFileProtocolException {
 
-        URL uiconLocation = file.getInformation().getIconLocation(IconDesc.SHORTCUT, iconSize,
+        URL uiconLocation = file.getInformation().getIconLocation(IconKind.SHORTCUT.getValue(), iconSize,
```

Comment:
We should pass the enum instead of the result of `getValue()`

Reply:
done